hack/lint.sh

```diff
@@ -2,8 +2,9 @@
 
 RETVAL=0
 GENERATED_FILES="zz_generated.*.go"
+GOLINT="${GOPATH}/bin/golint"
```

Comment:
Just to follow up to the commit-splitting, this *could* also have been 3 PRs:
* shellcheck fixes
* golint fix
* make verify-generated fix (OPERATOR_SDK)

I wouldn't insist on this, but it would allow unrelated topics to move independently, without one having to wait for the resolution of discussion about the other... :wink:

Reply:
> AFAICT, we don't expect or require GOPATH to be set since we're using go modules.
So how would you fix the issue, where would you take `golint` from

Reply:
What about the following:
> Maybe we can use the noobaa-operator way of doing go run golang.org/x/lint/golint ?